frontend/app/controllers/Healthcheck.scala

```diff
@@ -5,27 +5,36 @@ import play.api.mvc.{Action, Controller}
 import services.{TouchpointBackend, GuardianLiveEventService}
 import com.gu.monitoring.CloudWatchHealth
 import com.github.nscala_time.time.Imports._
+import play.api.libs.concurrent.Execution.Implicits._
 
-case class Test(name: String, result: () => Boolean)
+import scala.concurrent.Future
+
+case class Test(name: String, result: Future[Boolean])
```

Comment:
There's been a big change in semantics here - the tests are now only ever evaluated once!  :-1: